### ISSUE TITLE
Add COPM: COP Minteo

### ISSUE THREAD

**Issue by 0xnook** [celo-org/celo-token-list#62]
```
{
      "name": "COP Minteo",
      "address": "0xC92E8Fc2947E32F2B574CCA9F2F12097A71d5606",
      "symbol": "COPM",
      "decimals": 18,
      "chainId": 42220,
      "logoURI": "https://celo-org.github.io/celo-token-list/assets/minteo_copm.png"
}
```

#61

**Comment by 0xnook** [comment]
For future reference here's the tracker screenshot
https://celoscan.io/token/0xC92E8Fc2947E32F2B574CCA9F2F12097A71d5606
![image](https://github.com/user-attachments/assets/09aa5579-87e2-4774-b1b3-d526e26222d6)

**Comment by 0xnook** [comment]
Adding some context:

COP Minteo is a 1:1 fully fiat backed Colombian peso stablecoin, we deployed recently on Celo, and have been live in Polygon and Solana for around a year.

The new celo contract address, is already published in our official website [Minteo transparency](https://transparency.minteo.com/contract-directory)

**Comment by martinvol** [comment]
closed by #61
